### DISCUSSION TITLE
Is it possible to set a limit on the number of nodes allowed in a ZeroTier network?

### DISCUSSION THREAD

**Discussion by chonglee123456** [sinamics/ztnet#547] [Ideas]
Can we increase the limit on the number of nodes under each network ID

**Comment by tinola** [comment]
Can you clarify your question? What limit are you reaching? Any logs, screenshots?...

**Comment by chonglee123456** [comment]
Hello, what I mean is whether it is possible to limit the number of hosts or nodes under each network ID, such as limiting the number of access hosts under a certain ID to 20. thank you

**Reply by sinamics** [reply]
Unfortunately, that is not possible.

**Comment by mac-nofxmac** [comment]
Late answer, maybe useful to others having the same question though.
you can always set the size (approximately) of your networks by going to Ip Assignment , Advanced and set  a range of ips.
If you set multiple networks in close range and nodes are connected on more than one of those networks managed routes may conflict with each other. Since zerotier cannot prevent conflicts, is up to the administrator to prevent them.
An ip calculator is a helpful tool.. one of the many.. https://jodies.de/ipcalc?host=&mask1=29&mask2=
